config/esp32/components/chip/Kconfig

```diff
@@ -516,6 +516,51 @@ menu "CHIP Device Layer"
 
     endmenu
 
+    menu "Ethernet Options"
+        visible if ENABLE_ETHERNET_TELEMETRY
+
+        config GPIO_RANGE_MIN
+            int 
+            default 0
+
+        config GPIO_RANGE_MAX
+            int 
+            default 33 if IDF_TARGET_ESP32
+            default 46 if IDF_TARGET_ESP32S2
+            default 19 if IDF_TARGET_ESP32C3
+            default 48 if IDF_TARGET_ESP32S3
+
+        config ETH_MDC_GPIO
+            int "SMI MDC GPIO number"
+            range GPIO_RANGE_MIN GPIO_RANGE_MAX
+            default 23 if ENABLE_ETHERNET_TELEMETRY 
```

Comment:
Why is this `if ENABLE_ETHERNET_TELEMETRY` required here and at other places here?

Reply:
If not included .. these config options will be generated in other cases too when the option is not enabled

Reply:
But the usage of these config options in the code is already guarded under the ethernet flag, no?

Reply:
Yes they will not be used but still they will be generated in sdkconfig

Reply:
Ok. So, in default cases none of the ethernet options are seen in the sdkconfig. You've confirmed that, right?

Reply:
yes